fixmydjango/fixmydjango
# Add simple comment system

How to test:
- Install the new dependencies in the requirements.txt file
- Enter in any error page without authentication, you should not be able to see the comment form
- Enter in any error page with authentication, you should be able to see the comment form
- Try submit a comment with just plain text or with code

Comment section right beneath the How to reproduce and How to fix section:
![comment_section](https://cloud.githubusercontent.com/assets/8882260/23078017/cc17ed94-f525-11e6-86ee-a761b95ed32e.png)

Comment form right beneath the Comment section:
![comment_form](https://cloud.githubusercontent.com/assets/8882260/23078060/e7f8ccf4-f525-11e6-9c45-d309e25de849.png)
